rules/linux/command_and_control_frequent_egress_netcon_from_sus_executable.toml

```diff
@@ -0,0 +1,96 @@
+[metadata]
+creation_date = "2025/02/20"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "ES|QL rule type in technical preview as of 8.13"
+min_stack_version = "8.13.0"
+updated_date = "2025/02/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a high number of egress network connections from an unusual executable on a Linux system.
```

Comment:
I'd note something about this being an ES|QL rule and thus fields available in alert documents are limited. Reviewing the original docs related will aid in investigation.